### ISSUE TITLE
got two peaks in heatmap

### ISSUE THREAD

**Issue by YFan1997** [yezhengSTAT/CUTTag_tutorial#12]
Hello, Ye

I was following your tutorial and almost everything works well, but when I try to regenerate the heatmap, I got to peaks in my file, I am kind of doubt my hg38 tsv file, and was wondering whether other people happen to have the same issue with me.
Thank you!

Best,
Yifan
![matrix_gene_hg38](https://github.com/yezhengSTAT/CUTTag_tutorial/assets/147435992/18d7dc3d-547d-44ec-aa09-4c9be4a48807)

**Comment by yezhengSTAT** [comment]
I obtained my hg38.tsv table from https://genome.ucsc.edu/cgi-bin/hgTables. It looks like the following:

chr111868 14409 ENST00000456328.2 0 + 11868 11868 0 3 359,109,1189, 0,744,1352,
chr114403 29570 ENST00000488147.1 0 - 14403 14403 0 1198,34,152,159,198,136,137,147,99,154,37,0,601,1392,2203,2454,2829,3202,3511,3864,10334,15130,
chr117368 17436 ENST00000619216.1 0 - 17368 17368 0 1 68, 0,
chr129553 31097 ENST00000473358.1 0 + 29553 29553 0 3 486,104,122,0,1010,1422,
chr130365 30503 ENST00000607096.1 0 + 30365 30365 0 1 138,0,
chr134553 36081 ENST00000417324.1 0 - 34553 34553 0 3 621,205,361,0,723,1167,
chr152472 53312 ENST00000606857.1 0 + 52472 52472 0 1 840,0,
chr157597 64116 ENST00000642116.1 0 + 57597 57597 0 3 56,157,1201,0,1102,5318,
chr169054 70108 ENST00000335137.4 0 + 69090 70008 0 1 1054, 0,
chr189294 120932ENST00000466430.5 0 - 89294 89294 0 4 2335,150,105,158, 0,2796,23405,31480,
chr189550 91105 ENST00000495576.1 0 - 89550 89550 0 2 500,819,0,736,
chr1131024134836ENST00000442987.3 0 + 1310241310240 1 3812, 0,
chr1135140135895ENST00000494149.2 0 - 1351401351400 1 755,0,

**Comment by YFan1997** [comment]
Hello, Ye.

Thank you for your response, I used your data format (download from ucsc hg38 v44 known genome), and my file looks like this:
head hg38_v44_kg.bed
chr1	11868	14409	ENST00000456328.2	0	+	11868	11868	0,0,0	3	359,109,1189,	0,744,1352,
chr1	12009	13670	ENST00000450305.2	0	+	12009	12009	0,0,0	6	48,49,85,78,154,218,	0,169,603,965,1211,1443,
chr1	14403	29570	ENST00000488147.1	0	-	14403	14403	0,0,0	11	98,34,152,159,198,136,137,147,99,154,37,	0,601,1392,2203,2454,2829,3202,3511,3864,10334,15130,
chr1	17368	17436	ENST00000619216.1	0	-	17368	17368	0,0,0	1	68,	0,
chr1	29553	31097	ENST00000473358.1	0	+	29553	29553	0,0,0	3	486,104,122,	0,1010,1422,
chr1	30266	31109	ENST00000469289.1	0	+	30266	30266	0,0,0	2	401,134,	0,709,
chr1	30365	30503	ENST00000607096.1	0	+	30365	30365	0,0,0	1	138,	0,
chr1	34553	36081	ENST00000417324.1	0	-	34553	34553	0,0,0	3	621,205,361,	0,723,1167,
chr1	35244	36073	ENST00000461467.1	0	-	35244	35244	0,0,0	2	237,353,	0,476,
chr1	52472	53312	ENST00000606857.1	0	+	52472	52472	0,0,0	1	840,	0,

however, i still get a little bit different than yours, as mine heat map got different scale, even though its the same shape: 
![0329test](https://github.com/yezhengSTAT/CUTTag_tutorial/assets/147435992/e1894a51-c66b-4c58-b0c7-26efbfa879bb)


I was wondering if there is other steps that we did different, did you use the sorted.rmDup.sam for the mapped.bam or have you filter the quality score 2 and then you use these qc controled sam files?

ps: after I got my combined matrix, it got 160 mb size, is it normal? 

Thank you so much for your help!

Best,
Yifan
